Source/Microsoft.Teams.Apps.CompanyCommunicator/Controllers/DraftNotificationsController.cs

```diff
@@ -50,9 +50,9 @@ public class DraftNotificationsController : ControllerBase
         /// <param name="notification">A new Draft Notification to be created.</param>
         /// <returns>A task that represents the work queued to execute.</returns>
         [HttpPost]
-        public async Task CreateDraftNotificationAsync([FromBody]DraftNotification notification)
+        public async Task<DraftNotification> CreateDraftNotificationAsync([FromBody]DraftNotification notification)
```

Comment:
For now, let's have this return just the ID and not newly created/saved entity and not the whole object. We could change the status code returned to "created" if you would like. (and, if changed to "created" make sure it doesn't break the front end)

Reply:
Addressed.